config/settings_schema.json

```
@@ -250,6 +250,35 @@
       }
     ]
   },
+  {
+    "name": "t:settings_schema.search_input.name",
+    "settings": [
+      {
+        "type": "header",
+        "content": "t:settings_schema.search_input.settings.header.content"
+      },
+      {
+        "type": "checkbox",
+        "id": "predictive_search_enabled",
+        "label": "t:settings_schema.search_input.settings.predictive_search_enabled.label",
+        "info": "t:settings_schema.search_input.settings.predictive_search_enabled.info",
+        "default": true
+      },
+      {
+        "type": "checkbox",
+        "id": "predictive_search_show_vendor",
+        "label": "t:settings_schema.search_input.settings.predictive_search_show_vendor.label",
+        "default": false
```

```suggestion
        "info": "t:settings_schema.search_input.settings.predictive_search_show_vendor.info",
        "default": false
```
Is this how you do it? 👼 (Designer learning the thingzzz)

---

It is how you do it ❤️